src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -57,17 +61,42 @@ public virtual void ConfigureServices(IServiceCollection services)
                 });
             }
 
+            PrometheusMetricsConfig prometheusConfig = new PrometheusMetricsConfig();
+            Configuration.Bind("PrometheusMetrics", prometheusConfig);
+            services.AddSingleton(Options.Create(prometheusConfig));
+
+            if (prometheusConfig.Enabled)
+            {
+                services.Add<PrometheusMetricsServer>()
+                    .Singleton()
+                    .AsService<IStartable>();
+
+                if (prometheusConfig.SystemMetrics)
+                {
+                    services.AddSystemMetrics();
+                }
+            }
```

Comment:
Not required for now, but IStartupModel(IConfiguration) allows you to move this type of code out